extension/src/background.ts

```diff
@@ -12,3 +18,13 @@ chrome.runtime.onMessage.addListener(
     return true;
   }
 );
+
+// Give the browserAction a reason to exist other than "Enable RGH on this domain"
+chrome.browserAction.onClicked.addListener(() => {
```

Comment:
this looks like it should maybe bet letting people add domains, but in my testing doesn't seem to work...

Reply:
Hi @jaymzh, oh yes, we were did not get it to work and were just saving progress, and did not close the MR. There seemed to be an issue that stopped us but I do not recall it at the moment. Thank you for taking a look at it nevertheless!